taraxa/state/state_transition/state_transition.go

```diff
@@ -101,6 +104,14 @@ func (self *StateTransition) BeginBlock(blk_info *vm.BlockInfo) {
 		misc.ApplyDAOHardFork(&self.evm_state)
 		self.evm_state_checkpoint()
 	}
+	if blk_n == 12000 { // increase genesis balances fork
```

Comment:
this is not good. It should be passed in config as this for example: `if self.chain_cfg.ETHChainConfig.IsDAOFork(blk_n)`